pootle/apps/pootle_statistics/models.py

```diff
@@ -25,6 +27,17 @@
 SIMILARITY_THRESHOLD = 0.5
 
 
+class SubmissionActions(Enum):
+    EDIT = 1
+    REVIEW = 2
+    REVERT = 3
+    SUGG_ADD = 4
+    SUGG_ACCEPT = 5
+    SUGG_REJECT = 6
```

Comment:
I suggest name all the actions like `VERB_WHAT`, for example `REJECT_SUGGESTION` or `EDIT_TRANSLATION`.